apps/docs/pages/guides/database/extensions/pg_cron.mdx

```diff
@@ -1,7 +1,7 @@
 import Layout from '~/layouts/DefaultGuideLayout'
 
 export const meta = {
-  id: 'pgcron',
```

Comment:
Renamed any files that did not match the existing name in `extensions.json` and added the appropriate redirect.